src/style/app.module.css

```
@@ -507,11 +557,19 @@
 
 .tableHeader {
-  background-color: var(--table-head-bg);
-  color: var(--table-header-color);
+  background-color: var(--grey-bg-color);
+  color: var(--black-color);
   font-size: var(--font-size-header);
 }
 
+.profileImage {
+  display: flex;
+  justifycontent: center;
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)